### PR TITLE
Added Laravel, React Tutorial

### PR DESCRIPTION
**Pull request by endrose**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

**Comment by endrose** [comment]
fixing

**Comment by endrose** [comment]
fixing

File: free-courses-en.md [hunk 1]
```diff
@@ -439,6 +439,7 @@
 * [Learn to Program: The Fundamentals](https://www.coursera.org/learn/learn-to-program) (Coursera)
 * [Learn to Program Using Python](https://www.edx.org/course/learn-program-using-python-utarlingtonx-cse1309x) (edX)
 * [Programming for Everybody](https://www.coursera.org/learn/python)
+* [Introduction to Scripting in Python Specialization](https://www.coursera.org/specializations/introduction-scripting-in-python) (coursera)
```
Comment (SethiPandi) [inline free-courses-en.md:442]: Please write the name of the Coursera with the correct capitalization.

```suggestion
* [Introduction to Scripting in Python Specialization](https://www.coursera.org/specializations/introduction-scripting-in-python) (Coursera)
```

File: free-courses-id.md [hunk 2]
```diff
@@ -99,6 +99,9 @@
 
 * [Belajar ReactJS Bahasa Indonesia](https://www.youtube.com/playlist?list=PLCZlgfAG0GXALZIcEe2t3XVuQ50JYbsbA) - Sekolah Koding
 * [ReactJS Tutorial](https://www.youtube.com/playlist?list=PLU4DS8KR-LJ03qEsHn9zV4qdhcWtusBqb) - Prawito Hudoro
+* [Tutorial React js Indonesia](https://www.youtube.com/watch?v=mDNj800bjWQ&list=PLx4zY98IACXiMJTmFk_hyBOg6P1lEWHso&ab_channel=TahuCoding) - Tahu Coding
```
Comment (SethiPandi) [inline free-courses-id.md:102]: * Please use a link from the playlist endpoint, not the watch endpoint.
* Please name the resource after the source.
* There is no need to specify the language, after all this entire file is for that language.

```suggestion
* [React.js Fundamental](https://www.youtube.com/playlist?list=PLx4zY98IACXiMJTmFk_hyBOg6P1lEWHso) - Tahu Coding
```

**Review by SethiPandi — CHANGES_REQUESTED** [review]
Please also ensure you have all lists and categories in alphabetical order, and then you always have _exactly_ two line breaks above a new section.

You also need to resolve merge conflicts!

You can get more guidance on the formatting and conventions of this repository by checking out the [contribution guide](https://www.github.com/EbookFoundation/free-programming-books/tree/master/CONTRIBUTING.md).

You can also click on the failed checks to get more information on any check failures. You'll be able to see exactly what was wrong and how to fix it!

**Pull request closed without merging**